src/components/BackButton/index.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import { withStyles } from 'material-ui/styles';
+import Button from 'material-ui/Button';
+import ChevronLeft from 'material-ui-icons/ChevronLeft';
+import Typography from 'material-ui/Typography';
+import { FormattedMessage, injectIntl } from 'react-intl';
+
+import styles from './styles';
+import { AppLocation, RouterPath } from '../../constants';
+
+@withRouter
+@injectIntl
+@withStyles(styles, { withTheme: true })
+@connect((state, props) => ({
+  appLocation: state.App.get('appLocation'),
+}), (dispatch, props) => ({
```

Comment:
@dwalintukan if we're not using the dispatch, we don't need to put it here :)

Reply:
gotcha, will remove in a later cleanup PR

Reply:
so just dont include the 2nd param?